README.md

```diff
@@ -225,85 +286,90 @@ To run a specific test, use:
 pytest /utils/tests/test_parsers.py
 ```
 
-If you have a pre-existing installation of `pytest` on your machine, you may need to invoke your system's version of
-`python`/`python3` in the command:
+For more instructions and examples, please
+read [pytest's documentation](https://docs.pytest.org/en/stable/).
 
-```shell
-python3 -m pytest -v
-```
+### Load Tests
 
-For more instructions and examples, please read [pytest's documentation](https://docs.pytest.org/en/stable/).
+There are several ways you can run load tests. You can run `locust` locally or use the deployed
+version. You can also use a local mymove server or the one deployed in the load test environment.
+This means you have three possible combinations:
 
-## Running Load Tests
+* Run `locust` locally against local mymove server
+* Run `locust` locally against load test env mymove server
```

Comment:
I might spell out loadtest.dp3.us here, I think it gets a little muddled with all the environment, host, and server terms being thrown around getting used.

Reply:
Does https://github.com/transcom/milmove_load_testing/pull/97/commits/1c32c1dd7eca887b8691dc4dce43b01ebf66f8fa help? Or do you mean some other way?

Reply:
Thanks that's exactly what I meant.